### PR TITLE
Support for dotnetcore using new Visual Studio 2017 csproj format

### PR DESCRIPTION
**Pull request by roederja2**
I see that @adamhathcock  also added support for Microsoft.Framework.Configuration in #28 . This should probably be added as well as a separate pull request.

I wasn't sure how your continuous integration system works with regards to how the nuget packages are built etc. This probably needs some tweaking since it changed in Visual Studio 2017.

### CODE REVIEW

**Comment by ayoung** [comment]
I would really like this to be merged. I was just looking into how I could use HOCON in .NET Core.

**Comment by roederja2** [comment]
At the moment you'll just have to build the package yourself. The maintainer of this project seems to be somewhat inactive unfortunately.

File: src/core/Hocon/Akka.Hocon.csproj [hunk 1]
```diff
@@ -1,10 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.4;net461</TargetFrameworks>
+    <TargetFrameworks>net45;netstandard1.4</TargetFrameworks>
```
Comment (sean-gilliam) [inline src/core/Hocon/Akka.Hocon.csproj]: please change this line to target both net45 and netstandard1.6 to reflect what is in the main repo.

File: src/core/Hocon/Akka.Hocon.csproj [hunk 2]
```diff
@@ -1,80 +1,10 @@
...
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
-  </Target>
-  -->
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.4;net461</TargetFrameworks>
+  </PropertyGroup>
```
Comment (sean-gilliam) [inline src/core/Hocon/Akka.Hocon.csproj]: please add the following after the closing `PropertyGroup`
```xml
<PropertyGroup>
    <Copyright>Copyright © 2013-2017 Akka.NET Team</Copyright>
    <Authors>Akka.NET Team</Authors>
    <PackageIconUrl>http://getakka.net/images/AkkaNetLogo.Normal.png</PackageIconUrl>
    <PackageProjectUrl>https://github.com/akkadotnet/akka.net</PackageProjectUrl>
    <PackageLicenseUrl>https://github.com/akkadotnet/akka.net/blob/master/LICENSE</PackageLicenseUrl>
</PropertyGroup>
```

File: src/core/Hocon/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -5,11 +5,6 @@
 // General Information about an assembly is controlled through the following 
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("Akka.Configuration")]
-[assembly: AssemblyDescription("")]
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("")]
-[assembly: AssemblyProduct("Akka.Configuration")]
 [assembly: AssemblyCopyright("Copyright Akka .NET  2015")]
```
Comment (sean-gilliam) [inline src/core/Hocon/Properties/AssemblyInfo.cs]: please remove the ```AssemblyCopyright```, ```AssemblyTrademark```, and ```AssemblyCulture`` attributes

**Comment by sean-gilliam** [comment]
Other than my few comments, it looks good to me.

File: src/core/Hocon/Akka.Hocon.csproj (hunk 1 above)
Comment (roederja2) [inline src/core/Hocon/Akka.Hocon.csproj]: Why do you want to target netstandard1.6 when it also works with 1.4 ? I think Microsoft recommend using the lowest possible version for maximum compatibility.

Comment (sean-gilliam) [inline src/core/Hocon/Akka.Hocon.csproj]: because that's what we're targeting in v1.3 branch (current) in the main akka.net repo

**Pull request merged**